env-check.sh

```diff
@@ -1,17 +1,20 @@
 #!/bin/bash
+set -e
 
 mydir=$(cd "$(dirname "$0")"; pwd)
 
 # enable dynamic debug logs for SOF modules
-DYNDBG="/etc/modprobe.d/sof-dyndbg.conf"
+dyn_dbg_conf="/etc/modprobe.d/sof-dyndbg.conf"
+sof_logger="/usr/bin/sof-logger"
```

Comment:
better to use `which `or `type` to find if we have sof-logger or other executable binary in our PATH
user may not put sof-logger in this DIR